saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -225,17 +224,14 @@ def clean_input(cls, info, order, data):
                     )
 
         data["order_lines"] = order_lines
-        data["gift_card_lines"] = cls.get_gift_card_lines(lines_ids)
+        data["gift_card_lines"] = cls.get_gift_card_order_lines(order_lines)
```

Comment:
we could just `get_gift_card_lines(order_lines)` and drop method from line 233. It usually seems a good idea to extract such things to class methods but since this is another function anyway I wouldn't mind to drop it, to keep the class bit more compact.

Reply:
True. :)